docs/guides/tutorials/getting_started/set_up_data_docs.rst

```diff
@@ -9,17 +9,79 @@ In the previous steps, when you executed the last cell in the Jupyter notebook,
 
 .. figure:: /images/validation_results.png
 
-The validation results page is *descriptive*: it describes how a specific batch of data (in this case, the data we profiled) actually looked when validated, by showing the *Observed Value* that was found in the data batch. The validation results page is **also** *prescriptive*: it shows you your normative *Expectation* in human-readable form. Comparing what was expected and what was observed lets the validation result provide a *Status* that tells you whether the observed value *passed* or *failed* the validation.
+The validation results page shows you the results of using your Expectation Suite to validate a batch of data. In this case, you see the results of validating the ``yellow_tripdata_sample_2019_01`` table. All Expectations were automatically generated by the ``scaffold`` method, which we will explain below.
 
-If you scroll down, you will once again find the Expectation we defined for our ``passenger_count`` column, this time in human-readable format: **"distinct values must belong to this set"**. We also see the observed value for this batch, which is exactly the numbers 1 through 6 that we expected. This makes sense, since we're developing the Expectation using the January data batch.
+If you scroll down, you will see all Expectations that were generated for the ``passenger_count`` column. This includes the Expectation we wanted: **"distinct values must belong to this set: 1.0, 2.0, 3.0, 4.0, 5.0, 6.0"**.
+
+We also see the **observed values** for this batch, which is exactly the numbers 1 through 6 that we expected. This makes sense, since we're developing the Expectation using the January data batch.
 
 .. figure:: /images/validation_results_column.png
 
 **Feel free to click around and explore Data Docs a little more.** You will find two more interesting features:
 
 #. If you click on the *Home* page, you will see a list of all validation runs.
-#. The *Home* page also has a tab for your Expectation Suites. The Expectation Suite pages show you the exclusively *prescriptive* view of your data.
+#. The *Home* page also has a tab for your Expectation Suites, which shows you the Expectations you've created without any validation results.
 
 For now, your static site is built and stored locally. In the last step of the tutorial, we'll explain options for configuring, hosting and sharing it.
 
-But first, in the next step, we will complete the Great Expectations workflow by showing you how to validate a new batch of data with the Expectation Suite you just created!
\ No newline at end of file
+.. _tutorials__getting_started__create_your_first_expectations__what_just_happened:
+
+How did we get those Expectations?
```

Comment:
If anyone has a more elegant explanation of what scaffold does, please suggest!! This was definitely a point of confusion for testers, so I added my own explanation, but would love input.

Reply:
Ok!